Example/Sources/Helper/ColorsView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ColorsView.swift
+//  Example
+//
+//  Copyright © 2020 Bottle Rocket Studios. All rights reserved.
+//
+
+import SwiftUI
+
+struct ColorsView: View {
+
+    // MARK: - Properties
+    @State private var colors: [Color]
+
+    // MARK: - Initializers
+    init(colors: [Color]) {
+        self.colors = colors
```

Comment:
Since colors is a `@State`, it should be initialized as such.
```suggestion
        _colors = State(initialValue: colors)
```